tests/database/04-lock-recover/suite.rc

```diff
@@ -12,6 +12,7 @@ locker:lock => TRUES:succeed-all => done
     [[locker]]
         script = """
 timeout 60 cylc-db-lock
+cylc message "$CYLC_TASK_CYCLE_POINT I have locked the public database file"
```

Comment:
Yeah, it's wrong; the square brackets are placeholder for cycle point with no offset, not literal.  It only worked because of cylc automatically completing non-completed outputs when the task finished (and this branch doesn't do that).